src/main/kotlin/app/extractors/Heuristics.kt

```diff
@@ -1071,8 +1071,12 @@ val HeuristicsMap = mapOf<String, (String) -> ExtractorInterface?>(
     "ux" to { _ ->
         CommonExtractor(Lang.XML)
     },
-    "v" to { _ ->
-        CommonExtractor(Lang.VERILOG)
+    "v" to { buf ->
+        if (buf.contains("Require")) { 
```

Comment:
Verilog files can contain "Require" too.

Reply:
@anatolystansler, also I see no 'Require' in keywords as well, http://www.xilinx.com/support/documentation/sw_manuals/xilinx11/ite_r_verilog_reserved_words.htm

Reply:
What about "Require" word in comments of verilog?